dependencies/analyses/README.md

```diff
@@ -14,7 +14,7 @@ Running such an analysis on libpod may look as follows:
 
 ```
 # 1) Build the Podman binary with `-work -a`.
-[libpod]$ BUILDFLAGS="-work -a" make podman
+[libpod]$ BUILDFLAGS="-work -a" make bin/podman
```

Comment:
I am not sure this line needs to change.  I would leave this simpler for the users.

Reply:
@rhatdan Thanks for the feedback. On this occasion, I think we may need to keep this change. I hope I am not misunderstanding something obvious.

**Here is my logic:**

A part of the issue we are trying to solve is that we don't want to always rebuild the binaries if they are present (technically here, I tweaked this to be: we don't want to rebuild the binaries if they are present and if it was no change in the source code since the last build).

To achieve this, I renamed the target `podman` and `podman-remote` to `bin/podman` and `bin/podman-remote`.

As we put the binaries inside of the `bin` directory, if we keep the target name as `podman`, `makefile` will search for `./podman` instead of `bin/podman`. As it does not see it, it will build it every single time.

With the target name `podman`:
```sh
[vagrant@localhost libpod]$ ll bin/
total 55256
-rwxrwxr-x. 1 vagrant vagrant 56579112 Dec 29 22:05 podman
[vagrant@localhost libpod]$ make podman
...
GO111MODULE=on go build -mod=vendor  -gcflags 'all=-trimpath=/home/vagrant/go/src/github.com/containers/libpod/locallibpod/libpod' -asmflags 'all=-trimpath=/home/vagrant/go/src/github.com/containers/libpod/locallibpod/libpod' -ldflags '-X github.com/containers/libpod/libpod/define.gitCommit=1d83018f25a433eedcedf02ccc5515a507edb382-dirty -X github.com/containers/libpod/libpod/define.buildInfo=1577657174 -X github.com/containers/libpod/libpod/config._installPrefix=/usr/local -X github.com/containers/libpod/libpod/config._etcDir=/etc -extldflags ""' -tags "   selinux systemd exclude_graphdriver_devicemapper seccomp varlink" -o bin/podman github.com/containers/libpod/cmd/podman
[vagrant@localhost libpod]$ ll bin/
total 55256
-rwxrwxr-x. 1 vagrant vagrant 56579112 Dec 29 22:08 podman
```

As you can see, despite being here with the most recent version, it rebuilt over and over. This is a part of the issue raised #4367. This was the root cause of `make && sudo make install` failing.

With the target name `bin/podman`:
```sh
[libpod]$ ll bin/
total 107092
-rwxrwxr-x. 1 vagrant vagrant 56579112 Dec 29 21:42 podman
-rwxrwxr-x. 1 vagrant vagrant 53076704 Dec 29 21:44 podman-remote

[libpod]$ make bin/podman
...
make: `bin/podman' is up to date.
```

Let me know if you still think I should change something and I will make the necessary corrections.

Reply:
If you make 

podman: bin/podman

Would that satisfy your requirement?

Reply:
Yes we could keep those alias indeed.